src/components/Tag/index.js

```diff
@@ -2,15 +2,22 @@ import React from 'react'
 import './styles.scss'
 import PropTypes from 'prop-types'
 import { Text } from '../Typography/index'
+import { Icon } from '../Icon/index'
+import { Block } from '../Block'
 
 export const TAG_TYPES = ['success', 'danger', 'disabled', 'progress', 'default', 'warning']
 /**
  * Tag component that is used to specify status of a task
  */
-export const Tag = ({ className = '', type, label, children, ...props }) => {
+export const Tag = ({ className = '', type, label, children, icon, svg, ...props }) => {
+  const SvgIcon = svg
   return (
     <span className={['sg contacto-tag', className, `contacto-tag--${type}`].join(' ')} {...props}>
-      <Text type={type === 'default' ? 'caption' : 'caption-capital'}>{label || children}</Text>
+      <Block display="flex" alignItems="center">
+        {icon && <Icon name={icon} size={14} className="tag-icon" />}
+        {svg && <SvgIcon />}
```

Comment:
you need to add svg like below
<Icon svg={svg} size={14} className="tag-icon" />